.travis.yml

```diff
@@ -9,57 +11,41 @@ jobs:
     env:
       - CAN_FAIL=true
   include:
+    - stage: prepare cache
+      name: "Prepare the Ruby cache"
+      script: true
+      before_script:
+        - gem install asciidoctor asciidoctor-diagram rouge
+
     # fail if asciidoctor encounters errors
     - stage: validate
       name: "Validate updated assemblies with Asciidoctor"
-      install:
-        - gem install asciidoctor
-        - gem install asciidoctor-diagram
-        - gem install rouge
+      language: ruby
```

Comment:
**nitpick:** Assuming we already specified cache for both RVM and PIP we can skip this line.